tests/bootstrap.php

```diff
@@ -21,6 +21,11 @@
     define('PAYEE_ID', getenv('PAYEE_ID'));
     define('VERSION', getenv('VERSION'));
     define('SwedbankPay\\Api\\Client\\VERSION', VERSION);
+
+    if (getenv('MERCHANT_TOKEN_MOBILEPAY') && getenv('PAYEE_ID_MOBILEPAY')) {
```

Comment:
Codacy found an issue: [The use of function getenv() is discouraged](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-php/pullRequest?prid=5056329)